packages/components/tests/dummy/app/styles/components/dummy-component-props.scss

```diff
@@ -49,15 +49,28 @@
         background-color: #e4e4e4;
       }
     }
+  }
 
-    .default {
-      display: inline-block;
-      padding: 0.125rem 0.25rem;
-      border-radius: 4px;
-      font-weight: 700;
-      color: #0c56e9;
-      background-color: #cce3fe;
-    }
+  // Indicators / Tags
+  .default,
+  .required {
+    border-radius: 4px;
+    font-weight: 700;
+    display: inline-block;
+    padding: 0.125rem 0.25rem;
+  }
+
+  .default {
+    color: #0c56e9;
+    background-color: #cce3fe;
+  }
+
+  .required {
+    background-color: pink;
+    background-color: var(--token-color-surface-critical);
+    border: 1px solid var(--token-color-border-critical);
+    color: var(--token-color-foreground-critical-on-surface);
+    font-size: 0.9rem; // Note: Duplicated from parent: .dummy-component-props
```

Comment:
yes, it's ok because we may want to fine tune the size to have optical balance with the rest of the text.

In that sense, I have noticed that, despite having a smaller font size, optically the "required" tag seems bigger than the text next to it. 
<img width="262" alt="screenshot_1673" src="https://user-images.githubusercontent.com/686239/179934318-f4aa8d31-e28e-4757-955c-fd7d14fbf53a.png">
and the reason is that the font-family is different. I suggest to use the same font family as the normal `<code>` element so visually they're more balanced:
<img width="237" alt="screenshot_1674" src="https://user-images.githubusercontent.com/686239/179935462-fc1b0650-ce30-4076-b078-04f3332d6d3c.png">

(and potentially reduce the vertical padding for the required, or risks to become too prominent)

Reply:
Thanks, I'll update the styles of the "required" tag to use the code element's default font.

Note: The HTML code element typically uses a mono-spaced font to differentiate text which is meant to represent code from other "normal" text content. I agree it does look more balanced visually to use the same font for the "required" tag where I included it though. (We may want to use a different layout position or design solution to represent "required" values instead of using a mono-spaced font but we could look at that more in a future "non-scrappy" doc site.)

Reply:
After updating the font-family I deleted the font-size I had set so that it would inherit the size from the immediate parent.